src/modules/mdm.module.ts

```diff
@@ -26,6 +28,8 @@ import { YieldRatesModule } from '@ukef/modules/yield-rates/yield-rates.module';
     PremiumSchedulesModule,
     SectorIndustriesModule,
     YieldRatesModule,
+    OrdnanceSurveyModule,
```

Comment:
Do we need to expose this here? E.g., InformaticaModule is used by CustomersModule but is only imported there and not in the MDM module